perception/traffic_light_fine_detector/package.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>traffic_light_fine_detector</name>
```

Comment:
This package is not very deeply depending on the yolox model.
So instead of creating multiple packages for the fine detector, I would prefer to creating only one package providing multiple models (yolox, ssd-mobilenet), which could be addressed in the future.